bot.js

```diff
@@ -88,6 +97,18 @@ client.on('message', async (msg) => {
 		return userinfo(msg, member, author)
 	}
 
+	if (msg.content.startsWith('$tzrefresh')) {
+		userroles = msg.member.roles.cache
+		elite_crew_id = "760665499330936922"
+		bossman_id = "553723628957728820"
+		if (userroles.find(r => r.id === elite_crew_id) || userroles.find(r => r.id === bossman_id)) {
```

Comment:
finding if a user has a role when you already have the id can be easier than this.
future me, find the code you used for that

Reply:
future me here.
checkcanvote.js has a good example of doing that on a mass scale, but a quick and dirty method is this:
`if (msg.member.roles.cache.has("760665499330936922") || msg.member.roles.cache.has("553723628957728820")) {
    console.log("never gonna give you up")
}`

Reply:
Tried that earlier, but it didn't seem to work properly. I'll give it another go.

Reply:
Changed in #102ea0b; cleaned up role check